src/layouts/LinkReport/LinksReport.tsx

```diff
@@ -0,0 +1,342 @@
+import {
+  Box,
+  BreadcrumbItem,
+  Center,
+  HStack,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+  VStack,
+} from "@chakra-ui/react"
+import { Badge, Breadcrumb, Button, Link } from "@opengovsg/design-system-react"
+import { Redirect, useParams } from "react-router-dom"
+
+import { useGetStagingUrl } from "hooks/siteDashboardHooks"
+import { useGetBrokenLinks } from "hooks/siteDashboardHooks/useGetLinkChecker"
+import { useRefreshLinkChecker } from "hooks/siteDashboardHooks/useRefreshLinkChecker"
+
+import { NoBrokenLinksImage } from "assets"
+import {
+  isBrokenRefError,
+  NonPermalinkError,
+  NonPermalinkErrorDto,
+  RepoError,
+} from "types/linkReport"
+import { useErrorToast } from "utils"
+
+import { SiteViewHeader } from "../layouts/SiteViewLayout/SiteViewHeader"
+
+const getBreadcrumb = (viewablePageInCms: string): string => {
+  /**
+   * There are four main types of pages
+   * 1. /folders/parentFolder/subfolders/childFolder/editPage/page.md -> parentFolder/childFolder/page
+   * 2. /folders/parentFolder/editPage/page.md -> parentFolder/page
+   * 3. /editPage/page.md -> Feedback Form
+   * 4. /resourceRoom/resourceRmName/resourceCategory/resourceCatName/editPage/page.md -> resourceRmName/resourceCatName/page
+   */
+  const paths = viewablePageInCms.split("/")
+  let breadcrumb = paths
+    .filter((_, index) => index % 2 === 0)
+    .slice(2)
+    .join(" / ")
+    .replace(/-/g, " ")
+  if (breadcrumb.endsWith(".md")) {
+    breadcrumb = breadcrumb.slice(0, -3)
+  }
+
+  return breadcrumb
+}
+
+export const LinksReportBanner = () => {
+  const { siteName } = useParams<{ siteName: string }>()
+  const { mutate: refreshLinkChecker } = useRefreshLinkChecker(siteName)
+  const onClick = () => {
+    refreshLinkChecker(siteName)
+  }
+
+  const { data: brokenLinks } = useGetBrokenLinks(siteName)
+
```

Comment:
Hmm by default, does a site have no errors if the error checker isn't run prior? e.g. for new sites

Reply:
hey you are right on this, there is no auto link checker, since it triggered manually  
Not doing this for this experiment since the ones that we are experimenting with are a select -hardcoded list, which we can control.
in the future if doesnt exist, be should create! 
even if we forget, this pr already has a recovery path as shown here: 

https://github.com/isomerpages/isomercms-frontend/assets/42832651/929c93dd-28aa-4730-b3f9-c8d8a0695d53

Reply:
@alexanderleegs wah you caught me tripping off alarms. 

[2578d9a](https://github.com/isomerpages/isomercms-frontend/pull/1811/commits/2578d9a97dc0b45057d2a206ab71831e3300447d)
Fixing this issue upstream so that fe dont not ping be in the event of non feature flagged